shared/data/tracker_lists/trackersWithParentCompany.json

```diff
@@ -7190,269 +7190,74 @@
-            "c": "Facebook",
-            "u": "http://www.facebook.com/"
+            "u": "https://facebook.com",
+            "rules": [
+                {"rule": "facebook\\.com\\/ai\\.php\\?"},
+                {"rule": "facebook\\.com\\/audience_network\\/", "options": {"types": ["image"]}},
+                {"rule": "pixel\\.facebook\\.com($|[?/])"},
+                {"rule": "facebook\\.com.*\\/impression\\.php"},
+                {"rule": "graph\\.facebook\\.com\\/\\?ids=.*&callback=.*", "options": {"types": ["script"]}},
+                {"rule": "facebook\\.com\\/.*\\/plugins\\/send_to_messenger\\.php\\?app_id="},
+                {"rule": "facebook\\.com\\/brandlift\\.php"},
+                {"rule": "facebook\\.com\\/tr[?/]"},
+                {"rule": "facebook\\.com\\/common\\/scribe_endpoint\\.php"},
+                {"rule": "facebook\\.com\\/email_open_loc_pic\\.php"},
+                {"rule": "facebook\\.com\\/fr\\/u\\.php\\?"},
+                {"rule": "facebook\\.com\\/method\\/links\\.getStats\\?"},
+                {"rule": "facebook\\.com\\/offsite_event\\.php"},
+                {"rule": "facebook\\.com\\/rtb_impression\\/\\?"},
+                {"rule": "facebook\\.com\\/rtb_video\\/\\?"},
+                {"rule": "facebook\\.com[?/].*\\/tracking\\.js"},
+                {"rule": "facebook\\.com\\/ajax\/.*\\/log\\.php"},
+                {"rule": "facebook\\.com\\/ajax\\/.*logging\\."},
+                {"rule": "facebook\\.com\\/ct\\.php"},
+                {"rule": "facebook\\.com\\/friends\\/requests\\/log_impressions"},
+                {"rule": "facebook\\.com\\/search\\/web\\/instrumentation\\.php\\?"},
+                {"rule": "facebook\\.com\\/xti\\.php\\?"},
+                {"rule": "facebook\\.com[?/].*\\/impression_logging\\/"},
+                {"rule": "facebook\\.com[?/].*\\/instream\\/vast\\.xml\\?"}
+            ]
         },
         "facebook.net": {
             "c": "Facebook",
-            "u": "http://www.facebook.com/"
-        },
-        "fb.com": {
-            "c": "Facebook",
-            "u": "http://www.facebook.com/"
-        },
-        "atlassolutions.com": {
-            "c": "Facebook",
-            "u": "http://www.facebook.com/"
-        },
-        "friendfeed.com": {
-            "c": "Facebook",
-            "u": "http://www.facebook.com/"
-        },
-        "backtype.com": {
-            "c": "Twitter",
-            "u": "https://twitter.com/"
-        },
-        "crashlytics.com": {
-            "c": "Twitter",
-            "u": "https://twitter.com/"
-        },
-        "tweetdeck.com": {
-            "c": "Twitter",
-            "u": "https://twitter.com/"
-        },
-        "twimg.com": {
-            "c": "Twitter",
-            "u": "https://twitter.com/"
+            "u": "https://facebook.com",
+            "rules": [
+                {"rule": "connect\\.facebook\\.net\\/signals\\/"},
+                {"rule": "connect\\.facebook\\.net[?/].*\\/fbevents\\.js"},
```

Comment:
I might be missing something, but shouldn't `connect.facebook.net` always be marked as `Social` after this change?

Note this may be due to an issue elsewhere.

E.g. https://mashable.com/2009/01/12/facebook-connect-implementations/

![screenshot from 2018-06-21 14-53-05](https://user-images.githubusercontent.com/1431577/41724488-8cc2db4c-7565-11e8-8433-062c2f6873ba.png)

Reply:
It would need to go in TopTrackerDomains but there is probably a better way to handle that. I'll make a task to look into it.

Reply:
Ah, got it. Yeah, that view seems a bit incomplete otherwise.